client/ProductSizeModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+function ProductSizeModal(props) {
+  let sizes, image, isHidden;
+  sizes = !props.sizes
+    ? []
+    : props.sizes.map((size) => {
+      return (
+        <div className="product-dimensions__list-container" key={size.name + size.size + size.unit}>
+          <dt className="product-dimensions__list-item-name">
+            {size.name}:
+          </dt>
+          <dd className="product-dimensions__list-item-measure">
+            {size.size} {size.unit}
+          </dd>
+        </div>
+      );
+    });
+  image = props.image;
+  isHidden = props.isHidden
+    ? "seo-content"
+    : "seo-content unhide";
+
+  return (
+    <div className={isHidden}>
+      <div className="seo-content__modal-header">
+        <button
+          className="close"
+          onClick={props.toggleOverlay} >
+          <svg className="svg-icon close" focusable="false" viewBox="0 0 100 100" >
+            <path d="M20 20 L80 80 Z" stroke="black" stroke-width="10" fill="none" />
```

Comment:
React is telling me that stroke-width should be strokeWidth

Reply:
Yes, it should! Dashes don't work in React. Will fix, thanks!